nest/src/config.ts

```diff
@@ -0,0 +1,23 @@
+export const config = () => ({
+    census: {
+        serviceID: process.env.CENSUS_SERVICE_ID,
+        environment: 'ps2',
+        worlds: process.env.CENSUS_SUBSCRIPTION_WORLDS.split(',') ?? ['all'],
+    },
+
+    rabbitmq: {
+        urls: [process.env.RMQ_URL ?? 'amqp://localhost:5672'],
+        queue: process.env.RMQ_QUEUE ?? 'aggregator',
```

Comment:
This will need to be defined per environment, e.g. `aggregator-development` etc so we're able to split out on the rabbit server

Reply:
I think you can load in different configs based on the environment, but I think it would be easier to just have two different environment files for different environments.

Reply:
True, but the end result in rabbit we have `adminWebsocket-<env>` and `aggregator-<env>`